src/self_driving/pilot/action/change_knob_action.cpp

```diff
@@ -10,7 +10,9 @@ const std::string &ChangeKnobAction<T>::GetSQLCommand() {
   sql_command_ = "SET " + param_name_ + " ";
```

Comment:
Random thought: Should we have a strong typedef for SQL string instead of just using `std::string`?

Reply:
Sounds reasonable. I can add that.

Reply:
Eh I tried that but our strong typedef does not support string types: https://github.com/cmu-db/noisepage/blob/master/src/include/common/strong_typedef.h#L79